### PR TITLE
Release notes for 1.26

### LINKED ISSUE #3753: Release notes for `1.26.x`
### Release Notes:

* Listennotes.com isn't recognized to save for playlist #3726
* [Follow up #3383] History tab doesn't match private tab theme when empty #3719
* Allow user to disable 'long press to add media to playlist' #3670
* Add to Brave Playlist toast is not showing on Odysee.com #3663
* Do not show videos saved in playlist under Downloaded videos #3633
* [hackerone] RSS feed URL schemes #3630
* iOS - remove PIN entry from Keychain on fresh install #3598
* Update Startpage icon #3584
* Opening PDF crashes in Brave release and Dev #3572
* Audio continues to play even after tab is closed #3544
* New Menu UI #3491
* Feed state is retained when source is removed and re-added #3468
* Shields tooltip shown when tab tray is opened #3461
* Moving between text fields causes search button to be hidden (eventually) #3454
* Implement NTP Tutorial Page #3389
* Shared link from different sites on the same tab is incorrect #3246
* Reduce font weight for info text in shields settings #3180
* [upstream/firefox-ios] download leaks #2984
* Cancel button is not hidden completely #2925
* Changing theme while ad notification causes weird theme issues #2053
* Closing a private tab switches theme from dark to light #1862
* Quick theme toggle doesn't update theme for download/history/bookmarks #1722
* iOS 13 Dark mode auto switch #1572
* Add support to Auto-Close Tabs #1514
* Browser Menus do not Respond to Light/Dark OS Theme Changes #1506
* Memory leak in TabManager restoreAllTabs. #1086

### PR DESCRIPTION
**Pull request by kjozwiak**
## Summary of Changes

This pull request fixes #3753

## Submitter Checklist:

- [ ] *Unit Tests* are updated to cover new or changed functionality
- [ ] User-facing strings use `NSLocalizableString()`

## Test Plan:

N/A

## Reviewer Checklist:

- [x] Issues include necessary QA labels:
  - `QA/(Yes|No)`
  - `release-notes/(include|exclude)`
  - `bug` / `enhancement`
- [ ] Necessary [security reviews](https://github.com/brave/security/issues/new/choose) have taken place.
- [ ] Adequate unit test coverage exists to prevent regressions.
- [ ] Adequate test plan exists for QA to validate (if applicable).
- [ ] Issue is assigned to a milestone (should happen at merge time).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,34 @@
...
+- Updated settings menu to display scrollable modal which will allow for more dynamic actions. ([#3491](https://github.com/brave/brave-ios/issues/3491))
+- Updated Startpage icon under onboarding and search engine settings. ([#3584](https://github.com/brave/brave-ios/issues/3584))
+- [Security] Fixed cookie persistence via download request in private mode as reported on Bugzilla by Muneaki Nishimura. ([#2984](https://github.com/brave/brave-ios/issues/2984))
+- [Security] Fixed cross-site scripting issue in RSS feed as reported on HackerOne by Muneaki Nishimura. ([#3630](https://github.com/brave/brave-ios/issues/3630))
+- Improved readability of text under “Brave Shields & Privacy” settings by reducing font weight. ([#3180](https://github.com/brave/brave-ios/issues/3180))
+- Fixed playlist videos appearing under “Downloaded Videos” via iOS storage settings. ([#3633](https://github.com/brave/brave-ios/issues/3633))
+- Fixed https://www.listennotes.com not being recognized by Playlist. ([#3726](https://github.com/brave/brave-ios/issues/3726))
+- Fixed “Add to Brave Playlist” toast not being displayed on https://odysee.com. ([#3663](https://github.com/brave/brave-ios/issues/3663))
```
Comment (srirambv) [inline CHANGELOG.md]: Should be `Added playlist support for Odysee.com` . This is a new site that we started supporting not fixing an existing one.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,34 @@
...
+- Added a new tab tutorial page on startup for Japan region. ([#3389](https://github.com/brave/brave-ios/issues/3389))
+- Updated settings menu to display scrollable modal which will allow for more dynamic actions. ([#3491](https://github.com/brave/brave-ios/issues/3491))
+- Updated Startpage icon under onboarding and search engine settings. ([#3584](https://github.com/brave/brave-ios/issues/3584))
+- [Security] Fixed cookie persistence via download request in private mode as reported on Bugzilla by Muneaki Nishimura. ([#2984](https://github.com/brave/brave-ios/issues/2984))
+- [Security] Fixed cross-site scripting issue in RSS feed as reported on HackerOne by Muneaki Nishimura. ([#3630](https://github.com/brave/brave-ios/issues/3630))
+- Improved readability of text under “Brave Shields & Privacy” settings by reducing font weight. ([#3180](https://github.com/brave/brave-ios/issues/3180))
+- Fixed playlist videos appearing under “Downloaded Videos” via iOS storage settings. ([#3633](https://github.com/brave/brave-ios/issues/3633))
+- Fixed https://www.listennotes.com not being recognized by Playlist. ([#3726](https://github.com/brave/brave-ios/issues/3726))
```
Comment (srirambv) [inline CHANGELOG.md]: This shouldn't be include. Original issue #3648 was to support the new site but was marked as exclude. This should be `Added playlist support for listennotes.com`

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,34 @@
...
+- Fixed audio continuing to play after tab has already been closed in certain cases. ([#3544](https://github.com/brave/brave-ios/issues/3544))
+- Fixed Brave News feed state is retained when source is removed and re-added. ([#3468](https://github.com/brave/brave-ios/issues/3468))
+- Fixed moving between text fields causes search button to be hidden. ([#3454](https://github.com/brave/brave-ios/issues/3454))
+- Fixed “Trackers & ads blocked on this page” tooltip being displayed when tab tray is opened in certain cases. ([#3461](https://github.com/brave/brave-ios/issues/3461))
+- Fixed incorrect link being used when using “Share” in certain cases. ([#3246](https://github.com/brave/brave-ios/issues/3246))
+- Fixed “Cancel” button next to URL bar not being completely dismissed in certain cases. ([#2925](https://github.com/brave/brave-ios/issues/2925))
+- Fixed “History” window using incorrect theme under private tab when empty. ([#3719](https://github.com/brave/brave-ios/issues/3719))
+- Fixed theming issues when changing theme via iOS “Control Center” while ad notification visible. ([#2053](https://github.com/brave/brave-ios/issues/2053))
```
Comment (srirambv) [inline CHANGELOG.md]: Can change this to `Fixed browser theme issue when changing device theme via control centre while an ad notification is shown`. Theming doesn't sound right

**Review by srirambv — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (kjozwiak) [reply CHANGELOG.md]: Done 👍

File: CHANGELOG.md (hunk 2 above)
Comment (kjozwiak) [reply CHANGELOG.md]: Done 👍

File: CHANGELOG.md (hunk 3 above)
Comment (kjozwiak) [reply CHANGELOG.md]: Done 👍

**Review by srirambv — APPROVED** [review]
++

**Comment by kjozwiak** [comment]
Thanks for the review/feedback @srirambv 👍 Much appreciated.

**Pull request merged**